requirements.in

```diff
@@ -1,7 +1,7 @@
 more-executors
 pushsource>=2.13.3
 pushcollector
-cloudimg
+cloudimg==1.2.0
```

Comment:
Please use >= and not ==.  If you use ==, then it will not be possible to install the next release of pubtools-ami in the same environment as any cloudimg version newer than 1.2.0.